src/styles/_catalog.scss

```diff
@@ -14,9 +14,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
-  }
+    transition-duration: 300ms, 300ms;
+    transition-property: color, transform;
```

Comment:
value 300ms is the same for all transitions, use var
also consider using shorter syntax, a bit simpler to read for which prop you're applying transition
`transition: color 300ms, transform 300ms;`